src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -120,17 +131,53 @@ def init_from_file(self) -> None:
             profile_jsonc = commentjson.load(fileobj)
 
         self.profiles = profile_jsonc.get("profiles", {})
+        self.schema_property = profile_jsonc.get("$schema", None)
         self.defaults = profile_jsonc.get("defaults", {})
 
+        if self.schema_property:
+            self.validate_schema(config_type, opt_in)
         # loading secure props is done in load_profile_properties
         # since we want to try loading secure properties only when
         # we know that the profile has saved properties
         # self.load_secure_props()
 
+    def validate_schema(
+        self,
+        config_type: str,
+        opt_in: Optional[bool] = True,
+    ) -> None:
+        """
+        Get the $schema_property from the config and load the schema
+
+        Returns
+        -------
+        file_path to the $schema property
+        """
+
+        path_schema_json, path_config_json = None, ""
+
+        if self._location and config_type in ("Project Config", "Global Config"):
+            path_config_json = "./zowe.config.json"
+        elif self._location and config_type in ("Project User Config", "Global User Config"):
+            path_config_json = "./zowe.config.user.json"
```

Comment:
I think this could be simplified to `path_config_json = f"./{self.filename}"`, since the `filename` property returns either "zowe.config.json" or "zowe.config.user.json" depending on the config type.

Reply:
Thanks for the review, anything else left for the changes :relaxed: